colcon-test/action.yaml

```diff
@@ -0,0 +1,134 @@
+name: colcon-test
+description: ""
+
+inputs:
+  rosdistro:
+    description: ""
+    required: true
+  target-packages:
+    description: ""
+    required: true
+  build-depends-repos:
+    description: ""
+    required: false
+  label:
```

Comment:
By the way, renaming this to `label-regex` might be nice. How do you feel about it?

Reply:
Changed e408392